src/bootlint.js

```diff
@@ -969,17 +969,26 @@ var LocationIndex = _location.LocationIndex;
              * If there are any lint warnings, one general notification message will be window.alert()-ed to the user.
              * Each warning will be output individually using console.warn().
              * @param {string[]} disabledIds Array of string IDs of linters to disable
+             * @param {object} [alertOpts] Options object to configure alert()ing
+             * @param {boolean} [alertOpts.hasProblems=true] Show one alert() when the first lint problem is found?
+             * @param {boolean} [alertOpts.problemFree=true] Show one alert() at the end of linting if the page has no lint problems?
              * @returns {undefined} Nothing
              */
-            exports.showLintReportForCurrentDocument = function (disabledIds) {
+            exports.showLintReportForCurrentDocument = function (disabledIds, alertOpts) {
+                alertOpts = alertOpts || {};
+                var alertOnFirstProblem = alertOpts.hasProblems || alertOpts.hasProblems === undefined;
+                var alertIfNoProblems = alertOpts.problemFree || alertOpts.problemFree === undefined;
```

Comment:
Some others do (#216), and it makes sense to me since using the bookmarklet is probably the common case for `showLintReportForCurrentDocument()`, and in that situation it's reassuring to have a convenient indicator that the bookmarklet ran properly.
For your dev use case, it's not even 1 line of JS to restore the old behavior.
I'll defer to @hnrch02 as to whether to consider this patch (& 690606bae6e5187bbd7dcd5a5c7ac85e3cc8148a) backwards-compatible or not.